.github/workflows/e2e.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         with:
           cmd: install
       - name: Add hosts
-        run: make dev-env-setup
+        run: sudo yarn fec patch-etc-hosts
```

Comment:
those are different levels of abstraction; we should change the yarn command inside `make dev-env-setup` instead